src/core/api/subscription.ts

```diff
@@ -0,0 +1,14 @@
+import { apiClient } from '~core/apiClientInstance';
+import type { CurrentSubscriptionInfo } from '~features/subscriptions/types';
+
+export async function getCurrentUserSubscription() {
+  try {
+    return await apiClient.get<CurrentSubscriptionInfo>(
+      '/current_user/subscription ',
```

Comment:
Do we actually have this endpoint? If not, there's no need to implement it just yet, you can use mock data to initialize the page